sktalk/corpus/parsing/parser.py

```diff
@@ -8,10 +7,10 @@ class InputFile(abc.ABC):
     def __init__(self, path: str) -> None:
         self._path = path
         self._metadata = {"source": path}
+        self._utterances = []
```

Comment:
If you implement my suggestion for the `utterances` property below, you need to set this to `None` in order to distinguish the 'not extracted yet' case from the 'empty utterances' case.

```suggestion
        self._utterances = None
```